ui/ui-frontend/projects/design-system/src/app/components/inputs/inputs.component.html

```diff
@@ -123,6 +123,38 @@ <h2>Repeatable</h2>
         ></vitamui-common-repeatable-input>
       </div>
     </div>
+    <div class="row mb-4">
```

Comment:
Pour être iso avec les lignes d'avant et après

```suggestion
        <div class="d-flex flex-column">
      <div class="row m-0 pl-3 pr-3 mt-4">
```